01_Core/week03/submissions/exercise 4/index.html

```diff
@@ -0,0 +1,72 @@
+<!doctype html>
+
+<html lang="en">
+<head>
+  <meta charset="utf-8">
+
+  <title>CSS Exercise 4</title>
+  <meta name="description" content="ITA CSS Layout Exercise">
+  <meta name="author" content="Mark Kroh">
+
+  <style>
+  	.container	{
+		height: 320px;
+		width: 320px;
+		position: relative;
+	  }
+	
+	.interior  {
+		height: 100px;
+		width: 100px;
+	}
+
+	.div1	{
+		background-color: red;
+		position: absolute;
+	}
+
+	.div2	{
+		background-color: orange;
+		position: absolute;
```

Comment:
Since all of these `divs` have the same `position` and `border` values, you could combine those like mentioned on the first toy problem and just break them out for the styles that are unique to the individual `divs`